### PR TITLE
Add authentication web with token for apis

### LINKED ISSUE #22: Public API: Authentication with token
- Use normal web token

### PR DESCRIPTION
**Pull request by Linzeur**
Add new controller named "SessionController" for authentication for apis
Add rspec respectly
Combine authentication normal with authentication for token
Resolve #22

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -62,8 +62,9 @@ end
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
 gem 'rspec-rails'
 gem 'devise'
-gem "faker"
+gem 'faker'
```
Comment (yummta) [inline Gemfile]: Thanks for these details

File: app/controllers/api/shipment_controlle.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+class Api::ShipmentController < ApplicationController
+
+  def show
+    render json: "hola"
```
Comment (yummta) [inline app/controllers/api/shipment_controlle.rb]: Are you sure that this has to pass to master?

File: Gemfile (hunk 1 above)
Comment (Linzeur) [reply Gemfile]: Don't worry, although it is not necessary, as I saw that all gem use the character ', I only wanted to homogenize all lines

File: app/controllers/api/shipment_controlle.rb (hunk 2 above)
Comment (Linzeur) [reply app/controllers/api/shipment_controlle.rb]: Oh, you're right, thanks!, it was only for mi test.

**Comment by yummta** [comment]
Please review your conflicts with the file app/controllers/shipment_controller.rb

**Comment by Linzeur** [comment]
Thanks for the observation, I will fix it

**Pull request merged**